src/sinks/elasticsearch/common.rs

```diff
@@ -276,29 +276,35 @@ async fn get_version(
     tls_settings: &TlsSettings,
     proxy_config: &ProxyConfig,
 ) -> crate::Result<usize> {
+    #[derive(Deserialize)]
+    struct Version {
+        number: Option<String>,
+    }
     #[derive(Deserialize)]
     struct ClusterState {
```

Comment:
This seems incorrectly named given the changes, I'm also not sure if we need an additional struct or just the `version.number`?

Reply:
I think this is right, at least the structure, since the version number is nested this way.

Reply:
Mhmm, yea - I don't see a name for that API, so maybe `Response` is fine?

Reply:
Thanks for the feedback! `response` was already taken so I renamed ClusterState to ResponsePayload. Couldn't find the name of that API either. If there's a more concise way to express the shape of that payload for serde, I'm all for it. I also added a debug!() log statement for the auto-detected API version.